app/views/sequences/show.html.erb

```diff
@@ -85,13 +87,54 @@
                   </td>
                   <td><%= entry.name %></td>
                   <td><%= link_to entry.taxon.name, "http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id="+entry.taxon.id.to_s, :target => "_blank" %></td>
-                  <td><%= raw(entry.ec_cross_references.map{|ec| "<a href='http://enzyme.expasy.org/EC/"+ec.ec_id+"' target='_blank'>"+ec.ec_id+"</a>"}.join(", ")) %></td>
+                  <td><%= raw(entry.ec_cross_references.map{|ec| "<a href='http://enzyme.expasy.org/EC/"+ec.ec_number+"' target='_blank'>"+ec.ec_number+"</a>"}.join(", ")) %></td>
                   <td><%= raw(entry.go_cross_references.map{|go| "<a href='http://amigo.geneontology.org/amigo/term/"+go.go_id+"' target='_blank'>"+go.go_id+"</a>"}.join(", ")) %></td>
                 </tr>
               <% end %>
             <% end %>
           </table>
         </div>
+        <div class="tab-pane" id="ec-tree">
+          <div class="tpa-tree" id="d3TreeView"></div>
+          <div id="buttons-second" class="buttons-second"></div>
+        </div>
+        <div class="tab-pane" id="ec-table">
+          <div class="sequence_table">
+          <% colors = Hash.new %>
+          <table id="ec-table" class="table table-condensed">
+            <thead>
+              <tr>
+                <%=@test1%>
+                <% pos = 0%>
+                <% while pos < (@ec_column_name.length-1) do %>
+                  <th><a onclick="toggle_column(<%= pos+1 %>)" href="#"><span class="classdesc"><%= @ec_column_name[pos] %></span><span class="glyphicon glyphicon-resize-horizontal" style="display: none"></span></a></th>
```

Comment:
We don't want inline JavaScript. Please move to the .js file


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3465) by @bmesuere on Mon Nov 09 2015 at 20:59._